crates/polars-io/src/parquet/read_impl.rs

```diff
@@ -608,6 +608,8 @@ impl BatchedParquetReader {
         if self.limit == 0 && self.has_returned {
             return if self.chunks_fifo.is_empty() {
                 Ok(None)
+            } else if self.chunks_fifo.len() < n {
```

Comment:
Can you remove the extra `else if` and write `self.chunk_fifo.drain(..std::cmp::min(n, self.chunk_fifo.len()))`?

Reply:
Done, thanks for suggestion!
I moved `min` into a separate line for slightly better readability.